README.md

```diff
@@ -30,11 +30,9 @@ We've tested the project with the following:
 ## How do I use it?
 The custom layout is enabled by the `spatial` tool detecting a [new-format spatialos.json](docs/reference/project-configuration.md) file. 
 
-`spatial alpha local launch` and `spatial alpha cloud launch` work differently for a flexible project. You only get access to the new versions of these commands when you are **either**:
-* inside a directory containing a new-format `spatialos.json`, **or** 
-* when referencing a new-format `spatialos.json` using the `--main_config` flag
-
-Both `spatial alpha local launch` and `spatial alpha cloud launch` have a new parameter, `--world`, which points to a world configuration file. This flag is required to launch a deployment, although by default a file named [`world.json`](docs/reference/world-configuration.md) in the current working directory will be used if you don't pass the flag in.
+To support the FPL, we have introduced three toolbelt commands: `spatial alpha local launch`, `spatial alpha cloud upload` and `spatial alpha cloud launch`. Those commands only work for FPL projects. In order to use them, you need to:
```

Comment:
```suggestion
To support projects using FPL, we have introduced three spatial commands under `alpha` subcommand: `spatial alpha local launch`, `spatial alpha cloud upload` and `spatial alpha cloud launch`. These commands only work for FPL projects. In order to use them, you need to:
```

Reply:
+1 to @angelahcliu's suggestion of `toolbelt` -> `spatial` (but use "spatial CLI" in full for clarity).

"under the `alpha` subcommand" also sounds good to me, if "subcommand" is a standard phrase. 

Two suggestions:
1. I'd start with "As part of the FPL..." rather than "To support..."
2. Do we need the sentence "These commands only work for FPL projects"? Would it not be clear already, from the first sentence (especially if you use my update) and also because you're in the FPL repo?

Reply:
Rephrased the sentence.